terraform-infra-approvals/ctsc-work-allocation.json

```diff
@@ -0,0 +1,5 @@
+{
+  "module_calls": [
+   {"source":  "git@github.com:hmcts/terraform-module-servicebus-namespace?ref=common-tag-team-name"}
```

Comment:
why? module changes should be merged to master and can be tested either locally or in sandbox without approval needed

Reply:
it was suggested i prove that my changes in the module are working first which only works in the environments when run through AAT? preview doesn't run the tf

Reply:
you can run it in demo, sandbox or locally.

aat isn't for testing infra changes